test-framework/junit5-component/src/main/java/io/quarkus/test/component/QuarkusComponentTestExtensionBuilder.java

```diff
@@ -34,6 +34,7 @@ public class QuarkusComponentTestExtensionBuilder {
     private final List<AnnotationsTransformer> annotationsTransformers = new ArrayList<>();
     private final List<Converter<?>> configConverters = new ArrayList<>();
     private boolean useDefaultConfigProperties = false;
+    private boolean useSystemConfigSources = false;
```

Comment:
I think that we need to set this default value to `true` as well, for the case where `QuarkusComponentTestExtension.builder()` is used...

Reply:
Frankly, given 3.25 is not released yet, I would be tempted to just backport the initial change as is.

But maybe I'm underestimating the consequences of this change?

Reply:
Thanks, sounds great!